modules/dynamic-plugin-localization.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin/overview-dynamic-plugin.adoc
+
+:_mod-docs-content-type: Procedure
+[id="dynamic-plugin-api_{context}"]
+= Translating messages with react-i18next
+
+The link:https://github.com/openshift/console-plugin-template[plugin template] demonstrates how you can translate messages with link:https://www.i18next.com/[react-i18next]. The `v1alpha1` `apiVersion` of `ConsolePlugin` supports `console.openshift.io/use-i18n` annotation.
+
+.Prerequisites
+* You must have the plugin template cloned locally.
+* You must be running the {product-title} web console in a container. You can use either Docker or Podman 3.2.0+.
+* You must have `oc` installed.
+
+.Procedure
+
+. Indicate the `ConsolePlugin` contains localization resources. If the annotation is set to `true`, the localization resources from the i18n namespace named after the dynamic plugin are preloaded. If the annotation is set to any other value or is missing on the `ConsolePlugin` resource, localization resources are not preloaded.
```

Comment:
This is no longer an annotation. `spec.i18n.loadType: Preload` enables preloading (although this is mentioned below so maybe we delete this paragraph?)